scripts/deploy-base.js

```diff
@@ -24,9 +25,9 @@ const logExec = (cmd) => {
 };
 
 const buildBase = async () => {
-  await logExec(`docker build -t ${BASE}:latest -t ${BASE}:${VERSION} ./base`);
-  await logExec(`docker push ${BASE}:latest`);
-  await logExec(`docker push ${BASE}:${VERSION}`);
+  await logExec(
+    `docker buildx build --quiet --push --platform ${TARGET_ARCH.join(',')} -t ${BASE}:latest -t ${BASE}:${VERSION} ./base`
```

Comment:
@joelgriffith this will require something like what I proposed over here https://github.com/browserless/chrome/issues/1393#issuecomment-963700105
Since QEMU is unable to get chrome or chromium up and running, and it's "required" by the postinstall scripts... It will be able to build the image for the native machine you're using but fail for the emulated one.

The other way is like this https://github.com/browserless/chrome/issues/1393#issuecomment-963655465 but quite nasty nowadays IMO 😆 

BTW, if we don't run puppeteer while building, would also allow the image to be buildable for ARMv7 (RPi's on 32-bit mode).